packages/terra-tabs/src/_CollapsedTabs.jsx

```diff
@@ -26,8 +28,9 @@ const propTypes = {
 
 const CollapsedTabs = (props) => {
   props.onTruncationChange(false);
+  const theme = React.useContext(ThemeContext);
   return (
-    <div className={cx('collapsed-tabs-container')}>
+    <div className={classNames(cx('collapsed-tabs-container', theme.className))}>
```

Comment:
You don't need to run this through both classnames and cx if you're not combining local css classes with prop provided css classes
```suggestion
    <div className={cx('collapsed-tabs-container', theme.className)}>
```